docs/plugins/api.rst

```diff
@@ -207,11 +223,21 @@ You can even define subcommands by creating `command groups <https://click.palle
     def command():
         pass
 
-    @click.command(help="I'm a plugin subcommand")
+    @command.command(help="I'm a plugin subcommand")
```

Comment:
Wow, I had no idea that this was possible! What I usually do is that I add `command.add_command` statements, like here: https://github.com/overhangio/tutor-xqueue/blob/fb9e666814c1bdf6d00f8b80f94508b0404d5315/tutorxqueue/plugin.py#L201
But now I forgot to add this `add_command` statement to the docs, so you solution is better.